src/pages/workspace/WorkspacesListPage.js

```diff
@@ -115,6 +120,7 @@ class WorkspacesListPage extends Component {
      * @returns {Array} the menu item list
      */
     getWorkspaces() {
+        const reimbursementAccountBrickRoadIndicator = !_.isEmpty(this.props.reimbursementAccount.errors) ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : '';
```

Comment:
The `reimbursementAccount` is not per workspace. There is a single one. All workspaces will appear with a red dot if there is an error in `reimbursementAccount.errors`

Reply:
Just to be sure, if you link a bank account with any workspace, it will be linked with all of them?

Reply:
That seems to be how it is currently modelled. You can have N workspaces, but can only have one `reimbursementAccount`, so it is shared across workspaces.